src/mon/MonMap.h

```diff
@@ -412,6 +412,14 @@ class MonMap {
   int write(const char *fn);
   int read(const char *fn);
 
+  /**
+   * update CephContext with a copy of the current mon addrs
+   *
+   * @param cct context (and associated config)
+   */
+#ifndef WITH_SEASTAR
```

Comment:
seastar+crimson seems OSD specific and further crimson OSD seems to have a concept of ConfigProxy only and no CephContext(? afaict). Hence I assume this change is not applicable for the same.

Reply:
cc @tchaikov what you like to do here?

Reply:
we have a minimal CephContext implementation just for a very small set of use cases. and we don't use librados, so i think it's fine if this is disabled for crimson.

but more importantly, i don't think `update_cct()` should belong to `MonMap`.